java/spacewalk-java.changes.cbosdo.ks-cancel

```diff
@@ -0,0 +1,2 @@
+- Remove initiate action and cobbler system profile when
+  cancelling autoinstallion (bsc#1219645)
```

Comment:
Shouldn't this have both bsc#1220259 and bsc#1219645 mentioned. IIUC this PR is fixing both issues, right?
```suggestion
  cancelling autoinstallion (bsc#1219645, bsc#1220259)
```

Reply:
in fact 1219645 should be replaced by the other one, I'll fix it